tests/pausing_test.go

```diff
@@ -465,7 +470,13 @@ var _ = Describe("[rfe_id:3064][crit:medium][vendor:cnv-qe@redhat.com][level:com
 
 			By("Verifying VMI was indeed Paused")
 			uptimeDiffAfterPausing := hostUptime() - grepGuestUptime(vmi)
-			Expect(uptimeDiffAfterPausing).To(BeNumerically(">", uptimeDiffBeforePausing+10), "uptime diff after pausing should be greater by at least 10 than before pausing")
+
+			// We subtract from the sleep time the deviation due to the low resolution of `uptime` (seconds).
+			// If you capture the uptime when it is at the beginning of that second or at the end of that second,
+			// the value comes out the same even though in fact a whole second has almost passed.
+			// In extreme cases, as we take 4 readings (2 initially and 2 after the unpause), the deviation could be up to just under 4 seconds.
+			// This fact does not invalidate the purpose of the test, which is to prove that during the pause the vmi is actually paused.
+			Expect(uptimeDiffAfterPausing-uptimeDiffBeforePausing).To(BeNumerically(">", sleepTimeSeconds-deviation), "uptime diff after pausing should be greater by at least 10 than before pausing")
```

Comment:
@fossedihelm the change looks reasonable, and the comment you've added explains the rationale very well.

Just a couple of nits:

* the message passed to `Expect()` is no longer accurate and should be updated;
* shouldn't the numerical comparison use `>=` instead of `>`? Right now a value of exactly 4 will result in the test failing, while I believe it should pass in that case.

Reply:
Sure! Thank you for pointing that out to me! Done

Reply:
Since the expected sleep time is calculated dynamically based on `sleepTimeSeconds` and `deviation`, it would be more future-proof if we could reflect that dynamic value in the error message instead of hardcoding 6. But, I don't expect this test to be changed very often, and when it is we can just update the error message again, so I'm okay with leaving things as they are :)

Reply:
Thanks, I opted for the dynamic message :) Thanks